src/main/java/com/google/sps/servlets/AddSchoolData.java

```diff
@@ -162,4 +167,31 @@ private void putNewEntities(Entity school, Entity course, Entity term, Entity pr
       throw e;
     }
   }
+
+  private Date findTermDate(String termName) {
+    String[] termList = termName.split(" ", 5);
+
+    int termYear = Integer.parseInt(termList[1]);
+    Calendar startDay = Calendar.getInstance();
+
+    if (termList[0].equals("Spring")) {
+      startDay.set(Calendar.MONTH, 2);
+      startDay.set(Calendar.DATE, 01);
+      startDay.set(Calendar.YEAR, termYear);
+    } else if (termList[0].equals("Summer")) {
+      startDay.set(Calendar.MONTH, 5);
+      startDay.set(Calendar.DATE, 01);
+      startDay.set(Calendar.YEAR, termYear);
+    } else if (termList[0].equals("Fall")) {
+      startDay.set(Calendar.MONTH, 8);
+      startDay.set(Calendar.DATE, 01);
+      startDay.set(Calendar.YEAR, termYear);
+    } else {
+      startDay.set(Calendar.MONTH, 11);
+      startDay.set(Calendar.DATE, 01);
+      startDay.set(Calendar.YEAR, termYear);
```

Comment:
Is it better style to create a map or something and make a function for this?

Reply:
Not sure what you mean by map, we should discuss it during our standup.

Reply:
The hashmap was causing me some trouble so I just simplified the code as best as I could for the time being. This was laura can test her stuff